### PR TITLE
Add cerebro-iTerm

### PR DESCRIPTION
**Pull request by FracturedLoop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@
 - [cerebro-osx-contacts](https://www.npmjs.com/package/cerebro-osx-contacts) - Cerebro plugin to search in OSx Contacts.app.
 - [cerebro-osx-define](https://www.npmjs.com/package/cerebro-osx-define) - Cerebro plugin to define entered text in OSx dictionary.
 - [cerebro-osx-eject](https://www.npmjs.com/package/cerebro-osx-eject) - Quickly eject mounted volumes in macOS.
+- [cerebro-iterm](https://www.npmjs.com/package/cerebro-iterm) - Execute any shell command from Cerebro search line in iTerm
```
Comment (lubien) [inline README.md]: Show have a "." at the end.

File: README.md [hunk 2]
```diff
@@ -72,6 +72,7 @@
...
 - [cerebro-osx-system](https://www.npmjs.com/package/cerebro-osx-system) - Cerebro plugin to automate some OSx system features.
```
Comment (lubien) [inline README.md]: Newly added items should be at the end of it's list.

**Comment by lubien** [comment]
Can you fix and redo the PR's commit?

**Comment by FracturedLoop** [comment]
Typo fixed

**Comment by lubien** [comment]
I also need you to put it at the end of it's section then squash everything in one commit.

**Pull request closed without merging**